src/geocoders/mapbox.js

```diff
@@ -49,6 +49,11 @@ export var Mapbox = L.Class.extend({
           for (var j = 0; j < (loc.context || []).length; j++) {
             var id = loc.context[j].id.split('.')[0];
             properties[id] = loc.context[j].text;
+
+            // Get country code when available
+            if (loc.context[j].short_code != 'undefined') {
```

Comment:
Use one of the following variants (yours does not detect undefined correctly):
1. `if (typeof loc.context[j].short_code !== 'undefined')`
2. `if (loc.context[j].short_code !== undefined)`
3. `if (loc.context[j].short_code)`

I prefer variant 3.

Reply:
Got it !  I always struggle with those 'undefined' in JS..